fem/bilinearform.hpp

```diff
@@ -324,14 +330,15 @@ class BilinearForm : public Matrix
    double InnerProduct(const Vector &x, const Vector &y) const
    { return mat->InnerProduct (x, y); }
 
-   /// Returns a pointer to (approximation) of the matrix inverse:  \f$ M^{-1} \f$
+   /// Returns a pointer to (approximation) of the matrix inverse:  \f$ M^{-1} \f$ (currently returns NULL)
    virtual MatrixInverse *Inverse() const;
 
-   /// Finalizes the matrix initialization.
+   /** @brief  Finalizes the matrix initialization if the ::AssemblyLevel is LEGACY.
+       THe matrix that gets finalized is different if you are using static condensation
+       or hybridization.*/
    virtual void Finalize(int skip_zeros = 1);
 
    /** @brief Returns a const reference to the sparse matrix:  \f$ M \f$
-
```

Comment:
Empty lines like these have specific meaning -- they separate the brief section from the detailed section of the doxygen comments.

Reply:
Ack, I'll put the spaces back in.  Thanks for catching that.